### PR TITLE
Add "run" command to run an npm script in each package

### PR DESCRIPTION
**Pull request by RickWong**

### CODE REVIEW

File: bin/lerna.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env node
 
 var args = process.argv.slice(2);
-if (args.length > 1) {
```
Comment (sebmck) [inline bin/lerna.js]: This looks good although I'd prefer if the `args` assertion was present when the command isn't `run`.

Comment (RickWong) [inline bin/lerna.js]: Hey thanks for the comment. How would you like to see this part? Perhaps the check could be moved to the individual command scripts?

Comment (RickWong) [inline bin/lerna.js]: √ Done what you literally said. I've also added an "ls" command that I used to test the too-many-arguments warning with :)

**Comment by patrickheeney** [comment]
This looks cool. I wonder if it could be a bit more flexible like `lerna run --- make test-ci` to support more than just `npm run`, although that would be a good default.

**Comment by hzoo** [comment]
Is this still WIP, needs review, or ready?

**Comment by RickWong** [comment]
@hzoo It's functionally ready. Not sure whether @kittens has more feedback.

**Comment by kittens** [comment]
LGTM. We should probably add some integration tests where we just run a bunch of commands and check that the console output is right too.

**Comment by thejameskyle** [comment]
Moving to #44

**Pull request closed without merging**